newton/_src/utils/import_usd.py

```diff
@@ -938,7 +855,7 @@ def define_joint_mode(dof, joint_desc):
             print("Gravity direction:", scene_desc.gravityDirection)
             print("Gravity magnitude:", scene_desc.gravityMagnitude)
         builder.gravity = -scene_desc.gravityMagnitude * linear_unit
-        axis = Axis.from_any(int(np.argmax(np.abs(scene_desc.gravityDirection))))
+        stage_up_axis = Axis.from_any(int(np.argmax(np.abs(scene_desc.gravityDirection))))
```

Comment:
This is a weird thing to call `stage_up_axis`... its usually invalid & or the opposite of up